CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
+### Fixed
+- `NullReferenceException` when retrieving GPU information
```

Comment:
replace with the following:

Release notes without an image tag caused a `NullReferenceException` when retrieving GPU information. See issue #151